lib/OpenQA/WebAPI/Controller/Running.pm

```diff
@@ -115,9 +114,9 @@ sub livelog {
     # logfile and send them to the client, plus a utility function to
     # close the connection if anything goes wrong.
     my $id;
-    my $close = sub {
+    local *doclose = sub {
```

Comment:
The Perl documentation for this is definitely not great. And while i can't think of a strong reason against using a local glob here, the fact that it might have side effects (like @jberger mentioned), does make it a worse choice than a lexical scalar imho.